Pod/vialer-pjsip-include/pjmedia-codec/config_auto.h

```diff
@@ -87,7 +87,7 @@
 
 /* OPUS codec */
 #ifndef PJMEDIA_HAS_OPUS_CODEC
-#define PJMEDIA_HAS_OPUS_CODEC 0
+#define PJMEDIA_HAS_OPUS_CODEC 1
```

Comment:
I don't think this is needed when you've added this flag already in config_site.h. Please try without and let me know if you can do without.